src/otx/core/metrics/pck.py

```diff
@@ -11,7 +11,7 @@
 from torch import Tensor
 from torchmetrics import Metric
 
-from otx.algo.keypoint_detection.utils.keypoint_eval import simcc_pck_accuracy
+from otx.algo.keypoint_detection.utils.keypoint_eval import keypoint_pck_accuracy
```

Comment:
Importing `otx.algo` from `otx.core` doesn't look good in general, is it possible to import this function into core?

Reply:
Thank you for the good suggestion. I have moved this into core at 5dba92ed7ab858ca8509c7acc2cbc7576fcefec1.